site/_includes/scripts.html

```diff
@@ -1,12 +1,10 @@
 <script src="{{ site.cdn.jquery }}" integrity="{{ site.cdn.jquery_hash }}" crossorigin="anonymous"></script>
 <script>window.jQuery || document.write('<script src="{{ site.baseurl }}/assets/js/vendor/jquery-slim.min.js"><\/script>')</script>
 
-<script src="{{ site.baseurl }}/docs/{{ site.docs_version }}/assets/js/vendor/popper.min.js"{% if site.github %} integrity="{{ site.cdn.popper_hash }}" crossorigin="anonymous"{% endif %}></script>
-
 {%- if site.github -%}
-  <script src="{{ site.baseurl }}/docs/{{ site.docs_version }}/dist/js/bootstrap.min.js" integrity="{{ site.cdn.js_hash }}" crossorigin="anonymous"></script>
+  <script src="{{ site.baseurl }}/docs/{{ site.docs_version }}/dist/js/bootstrap.bundle.min.js" integrity="{{ site.cdn.js_hash }}" crossorigin="anonymous"></script>
 {%- else -%}
-  <script src="{{ site.baseurl }}/docs/{{ site.docs_version }}/dist/js/bootstrap.js"></script>
+  <script src="{{ site.baseurl }}/docs/{{ site.docs_version }}/dist/js/bootstrap.bundle.js"></script>
```

Comment:
This seems like a less helpful if/else block now. Should we use the minified bundle here to match other views?

Reply:
Right, but we're building the minified bundle locally anyway, so why not use minified here? Is it for seeing the code easier or something while developing? In fact, why include non-minified files in our dist at all for that matter? :D

Reply:
Because it allows folks to minify our files the way they want 😉

Reply:
In theory, with maps in place debugging isn't an issue.

Reply:
I defer to you then @XhmikosR. I think we can use the minified one here, but I'm fine either way for now.

Reply:
I think it'd be better if we left this as is for now and we can revisit later. I mean, it certainly doesn't hurt and is basically only for people developing locally :)